ui/ember-cli-build.js

```diff
@@ -34,8 +34,9 @@ const appConfig = {
       return `${config.rootURL.replace(/\/$/, '')}${filePath}`;
     },
   },
-  babel: {
-    plugins: [['inline-json-import', {}]],
+  'ember-cli-babel': {
+    enableTypeScriptTransform: true,
```

Comment:
This replaces `ember-cli-typescript` and lets `ember-cli-babel` do the TS transforms.